lib/http_server_fixture/routes.ex

```diff
@@ -1,9 +1,11 @@
 defmodule HTTPServerFixture.Routes do
```

Comment:
I think you are missing an `end` to go with this `do` 👀

Reply:
I'm pretty sure the `do:` syntax doesn't require an `end` to go with it like a `do`-block does!

However, looking at the [documentation](https://elixir-lang.org/getting-started/modules-and-functions.html#named-functions) I found this quote:

> You may use `do:` for one-liners but always use `do`-blocks for functions spanning multiple lines. 

I guess now I'm wondering if the map is considered "one-line" as it's a single data structure, or if I need to change the `do:` to a `do`-block because of the way it's written!

Reply:
Oooh I wonder if that's because of the fact it seems multi-lined? I'll look into that a little more! This conversation reminded me to use `mix format` too and there was lot that needed to be updated so I'll push that up as well.